gradle.properties

```diff
@@ -7,6 +7,8 @@ parchment_version=2022.05.02
 fabric_loader_version=0.13.3
 fabric_version=0.51.1+1.18.2
 forge_version=1.18.2-40.1.0
+# DevAuth version
+devauth_version=1.0.0
 # Note: Must use same version of eventbus as is used in the above forge version!
 forge_eventbus_version=5.0.+
```

Comment:
Actually just noticed this, can we not separate the forge event bus and forge version? They are closely related and a change to one may cause the other one to be outdated as well.

Reply:
Agree. Other than that, it looks good.

Reply:
whoops